shared/elements/gaia_grid/js/grid_dragdrop.js

```diff
@@ -7,6 +7,16 @@
 
   const COLLECTION_DROP_SCALE = 0.5;
 
+  /* This is an alternative delay used to initiate icon movement when in edit
+   * mode. The normal contextmenu delay is too long once we're in edit mode
+   * and causes user confusion, but a short delay globally would cause too many
+   * accidental triggers of edit mode.
+   */
+  const EDIT_LONG_PRESS_DELAY = 100;
+
+  /* The movement threshold to use for the above synthetic long-press. */
+  const EDIT_LONG_PRESS_THRESHOLD = 5;
```

Comment:
Should we multiple this by ```window.devicePixelRatio```? I have a feeling it would be very easy to go outside of this threshold on a @2.25x and higher device.